examples/webgl_loader_xml_vtk.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
```

Comment:
I don't think you need to have 3 versions of the bunny, you can just load the bunny like the current example does, and then the new cubes.